### ISSUE TITLE
Use browserify for more modular approach

### ISSUE THREAD

**Issue by cirocosta** [pedronauck/react-simpletabs#4]
Hey @pedronauck , i've started doing some tests (finally got some time to tinker with react-simpletabs), what do you think of using browserify (i'd go with webpack, but the project already uses gulp ..) to make everything more moduler and then easier to test? It'd just require configuring it for being gracefully with UMD pattern (i don't know how browserify deals with builds for libraries, do you?) and using React as an external (so that the user is able to require it from its node_modules/cache if using in a commonjs scenario, or requiring it .. anyway).

**tldr**, IMO letting everything modular from the beginning is the right approach. What do you think?
